src/parts/_misc.css

```diff
@@ -54,3 +54,41 @@ tbody tr:nth-child(even) {
 ::selection {
   background-color: var(--selection);
 }
+
+details {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  background-color: var(--background-alt);
+  padding: 10px 10px 0;
+  margin: 1em 0;
+  border-radius: 6px;
+  overflow: hidden;
+}
+
+details[open] {
+  padding: 10px;
+}
+
+details > *:last-child {
```

Comment:
Can you change this to `details > :last-child`? We would want to avoid `*` selectors where possible.

Reply:
Done!